wrappers/python/tests/layer/test_attention.py

```diff
@@ -292,7 +292,7 @@ def test_dynamic(
     )
     attn.init_randn_async()
 
-    out_dynamic_actual = attn.forward_dynamic(inp_tm)
+    out_dynamic_actual, _ = attn.forward_dynamic(inp_tm)
```

Comment:
Could you please add a test, that running a prefill with n-2 tokens and 2 subsequent decodes generates nearly the same tensor of logits as if a single decode with n tokens?